docs/geedocs/answer/6259403.html

```diff
@@ -10,6 +10,7 @@
 <!-- 6259403.html -->
 <div class="container">
 <div class="sidebar1">
+  <p class="sidebar-toc"><a href="../answer/7160000.html">Open GEE 5.2.0</a></p>
```

Comment:
Side note: I wonder if there's a way that we could create include files for these menus using Jekyll. Another option might be to use javascript or go real old-school and use iframes :)